salt/example.top

```diff
@@ -4,7 +4,9 @@ base:
         - elife.nginx
         - elife.nodejs6
         - elife.python3
+        - elife.postgresql
         - elife.aws-cli
         - elife.external-volume
         - elife.external-volume-srv
+        - reviewer-suggestions/postgres
```

Comment:
I didn't know about the `/` notation, is it different from the `.` as a separator?